composer.json

```diff
@@ -55,6 +55,9 @@
             "dev-master": "2.1-dev",
             "dev-develop": "2.2-dev",
             "dev-release-3.0.0": "3.0.x-dev"
+        },
+        "zf": {
+            "config-provider": "Zend\\Expressive\\Router\\ConfigProvider"
```

Comment:
Maybe we should consider change namespace in the package to have it consistent with repository name.
Now it is so confusing, and we are using the same namespace in coupe repositories...

Reply:
Doing so would be quite problematic from a migration stand-point, however.

That said: @xtreamwayz We should likely put the `ConfigProvider` classes for the routers and template providers into a subnamespace in order to prevent collisions: `Zend\Expressive\Router\FastRoute\ConfigProvider`.